styles.css

```diff
@@ -0,0 +1,160 @@
+body {
+  /* background: url(images/bg.jpg); */
+  height: 88vh;
+  color: #000;
+  background-repeat: no-repeat;
+  background-size: cover;
+  text-shadow: 1px 2px 3px #000;
+}
+
+#date {
+  display: flex;
+  justify-content: flex-end;
+  padding: 5px 10px;
+}
+
+.library-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-family: "Courier New", Courier, monospace;
+  margin-top: 10%;
+}
+
+.headline {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+}
+
+.danger {
+  color: red;
+  padding-top: 10px;
+  font-size: 20px;
+}
+
+.success {
+  color: green;
+  padding-top: 10px;
+  font-size: 20px;
+}
+
+.display-books-container {
+  display: flex;
+  flex-direction: column;
+  width: 60%;
+  height: 40%;
+  max-height: 100%;
+  border: 20px solid rgba(11, 175, 230, 0.641);
+  padding: 10px;
+  border-radius: 5px;
+  box-shadow: 1px 2px 3px rgb(0, 0, 0);
+  margin-top: 10px;
+}
+
+.display-book {
+  display: flex;
+  justify-content: space-between;
+  padding: 2px;
+  width: 100%;
+}
+
+.display-book-details {
+  display: flex;
+  align-items: center;
+  margin-right: 6px;
+  background: rgba(55, 55, 216, 0.592);
+  padding: 0 3px 0 3px;
+  width: 100%;
+  color: rgb(254, 254, 254);
+  border: 1px solid white;
+  border-radius: 5px;
+  box-shadow: 1px 2px 4px black;
+  text-shadow: 1px 2px 4px rgb(5, 5, 5);
+}
+
+.display-book button {
+  background: rgba(55, 55, 216, 0.592);
+  color: rgb(254, 254, 254);
+  border: 1px solid white;
+  border-radius: 5px;
+  box-shadow: 1px 2px 3px black;
+  text-shadow: 1px 2px 3px rgb(5, 5, 5);
+}
+
+.input-container button {
+  padding: 10px 20px;
+  background: rgb(55, 55, 216);
+  color: rgb(254, 254, 254);
+  border: 1px solid white;
+  border-radius: 5px;
+  box-shadow: 1px 2px 4px black;
+  text-shadow: 1px 2px 4px rgb(5, 5, 5);
+}
+
+.display-book button:active {
+  background: rgba(250, 109, 15, 0.592);
+}
+
+.underline {
+  margin-top: 20px;
+  width: 20%;
+}
+
+.input-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 50%;
+}
+
+/* .alert-container {
+  background-color: red;
+} */
```

Comment:
## Recommendation Change
I am asking for this change because you did well in the project as required but I highly recommend you try not to leave commented code on your codebase as it makes your code not professional. It's a tiny mistake and I encourage you not to have commented code in your codebase in all your future projects. I might ask for changes next time or another reviewer. Please remove this commented code 👍🏽

Reply:
done, thanx